### DISCUSSION TITLE
Show windows border

### DISCUSSION THREAD

**Discussion by Wateir** [kovidgoyal/kitty#8065] [Q&A]
When kitty have only one windows, i want to show the border of the windows.

Currently i have it when i have 2 or more windows open

**Comment by kovidgoyal** [comment]
Search for must_draw_borders

**Comment by Wateir** [comment]
It's not on the wiki ?

![image](https://github.com/user-attachments/assets/7fb025b5-c67e-4a7f-bb15-3289f283d9db)

maybe i don't have kitty up to date ? it's a new feature ?

**Comment by Wateir** [comment]
So it's not the update, i have the last version of kitty on the Arch repo

**Comment by kovidgoyal** [comment]
I didnt say it was config setting I said search for it, in the issue tracker you will find how to use it.

**Comment by Wateir** [comment]
https://github.com/kovidgoyal/kitty/blob/cb2e1fcd042b48a0f669f7d37cfbdc098773b1ad/kitty/layout/base.py#L221

so i need to make this line to true, and recompile all kitty by myselft ?

**Comment by kovidgoyal** [comment]
No, you need to search the kitty issues/discussion for that word and
find where someone has laid out how to use it for you.

On Sun, Nov 24, 2024 at 03:33:16AM -0800, Wateir wrote:
> https://github.com/kovidgoyal/kitty/blob/cb2e1fcd042b48a0f669f7d37cfbdc098773b1ad/kitty/layout/base.py#L221
>
> so i need to make this line to true, and recompile all kitty by myselft ?
>
> --
> Reply to this email directly or view it on GitHub:
> https://github.com/kovidgoyal/kitty/discussions/8065#discussioncomment-11362616
> You are receiving this because you commented.
>
> Message ID: ***@***.***>

--
_____________________________________

Dr. Kovid Goyal
https://www.kovidgoyal.net
https://calibre-ebook.com
_____________________________________
